python/lib/sift_py/grpc/transport.py

```diff
@@ -126,25 +138,35 @@ def _compute_channel_options(opts: Optional[SiftChannelConfig] = None) -> List[T
     return options
 
 
-def _metadata_interceptor(config: SiftChannelConfig) -> ClientInterceptor:
+def _metadata_interceptor(
+    config: SiftChannelConfig, metadata_kwargs: Optional[Dict[str, Any]] = None
```

Comment:
nit: I think I'd rather call this `metadata` just to be consistent with other functions and rename the variable to `md`?